src/expo/expo_client/get.rs

```diff
@@ -46,35 +33,49 @@ pub(crate) async fn get_push_notification_receipts(
     {
         Ok(response) => {
             if response.status().is_success() {
-                let result: PushResult = response.json::<PushResult>().await.map_err(|err| {
-                    CustomError::DeserializeErr(format!("Failed to deserialize response: {}", err))
-                })?;
+                let result = response
+                    .json::<GetPushNotificationReceiptsResponse>()
```

Comment:
`/api/v2/push/getReceipts ` will return 
```
{
  "data": {
    Receipt ID: {
      "status": "error" | "ok",
      // if status === "error"
      "message": string,
      "details": JSON
    },
    ...
  },
  // only populated if there was an error with the entire request
  "errors": [{
    "code": string,
    "message": string
  }]
}
```
like this.
So the response should be array.

Reply:
I am gonna fix it. @katayama8000

Reply:
https://docs.expo.dev/push-notifications/sending-notifications/#push-receipt-response-format

Reply:
response should not be array.... 😢